packages/client/hmi-client/src/workflow/tera-simulate-chart.vue

```diff
@@ -40,8 +46,14 @@ const props = defineProps<{
 	initialData?: CsvAsset;
 	mapping?: { [key: string]: string }[];
 	runType?: RunType;
+	size?: { width: number; height: number };
 }>();
 
+const chartSize = computed(() => {
+	if (props.size) return props.size;
+	return { width: 390, height: 190 };
+});
```

Comment:
Why not a `reactive` which could make sure both values are set and have a default?

```suggestion
const chartSize = reactive({
    width: props.size?.width ?? 390,
    height: props.size?.height ?? 190
});
```

Reply:
I am a bit confused - `reactive` does not guarantee whether values should exist or not. But I want to keep sizing static where possible, for now, because that was partial to a lot of graph-jittering problems with chart.js.

Reply:
Reactive was just a suggestion, no need to use it.
I wanted to say, why not testing that both value are set properly while at it?

```suggestion
const chartSize = computed(() => {
	return { 
		width: props?.size?.width ?? 390, 
		height: props?.size?.height ?? 190 
	};
});
```

this is a trivial question, not worth spending time about it. Sorry for the confusion.